compute/api/create_instance.py

```diff
@@ -154,7 +153,7 @@ def main(project, bucket, zone, instance_name, wait=True):
 
     instances = list_instances(compute, project, zone)
 
-    print('Instances in project %s and zone %s:' % (project, zone))
+    print('Instances in project {} and zone {}:'.format(project, zone))
```

Comment:
I guess if we run this next time, it will transform to f-string :D 

%s -> format() -> f-string can happen only one step at a time ;)

Reply:
I fixed this manually.